docs/reference/snapshot-restore/repository-s3.asciidoc

```diff
@@ -329,6 +329,20 @@ include::repository-shared-settings.asciidoc[]
     `1000` which is the maximum number supported by the https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListMultipartUploads.html[AWS
     ListMultipartUploads API]. If set to `0`, {es} will not attempt to clean up dangling multipart uploads.
 
+`throttled_delete_retry.delay_increment`::
+
+    (integer) This value is used as the delay before the first retry and the amount the delay is incremented by on each subsequent retry. Default is 50ms, minimum is 0ms.
+
+`throttled_delete_retry.maximum_delay`::
+
+    (integer) This is the upper bound on how long the delays between retries will grow to. Default is 500ms, minimum is 0ms.
```

Comment:
Likewise

```suggestion
    (<<time-units,time value>>) This is the upper bound on how long the delays between retries will grow to. Default is 500ms, minimum is 0ms.
```

Reply:
Fixed in 1e207e0ce1eb32271da675bde27368eadafab953